web/styles/zulip.css

```diff
@@ -1695,6 +1695,12 @@ div.focused_table {
 
     &.condensed {
         max-height: 8.5em;
+        /* Narrower screens need a bit more preview space because there is less space.
+           This is also important for link previews, since otherwise they would get cut
+           off halfway through the image. */
```

Comment:
This isn't quite right -- I'd need to also change the logic about when to condense, because at certain heights the more/less will show without it doing anything (because it actually is tall enough with this extended height).

Reply:
Does this seem like the right direction to go, though?